docs/src/quickstart.md

```diff
@@ -160,6 +161,51 @@ plot_energy(idata);
 gcf()
 ```
 
+### Additional information in Turing.jl
+In the above `from_mcmcchains` call, we did not specify the `prior`, `posterior_predictive`, nor the `log_likelihoods`. This section describes how one would go about obtaining these from the `Model` and `Chains`, which we can then pass to `from_mcmcchains` to construct the [`InferenceData`](@ref).
+
+To sample from the prior, one simply calls `sample` but with the `Prior` sampler:
+```@example quickstart
+prior = sample(param_mod, Prior(), nsamples; progress = false)
+```
+Obtaining the predictive posterior is a matter of first instantiating a "predictive model", i.e. a `Model` but with the observations set to `missing`, and then call `predict` on the predictive model and the inferred posterior:
+```@example quickstart
+# Instantiate the predictive model
+param_mod_predict = turing_model(
+    J, fill(missing, length(y)), σ
+)
+# and then sample!
+posterior_predictive = predict(param_mod_predict, turing_chns)
```

Comment:
This should work, right?
```suggestion
prior_predictive = predict(param_mod_predict, prior)
posterior_predictive = predict(param_mod_predict, turing_chns)
```

Reply:
Forgot about the prior predictive; added!